tests/phpunit/Mutator/AbstractMutatorTestCase.php

```diff
@@ -76,8 +75,14 @@ abstract class AbstractMutatorTestCase extends TestCase
      */
     private static $printer;
 
+    /**
+     * @var MutatorFactory|null
+     */
+    private static $mutatorFactory;
+
     protected function setUp(): void
     {
+        // TODO: refactor this bit...
         $this->mutator = $this->createMutator();
```

Comment:
I need to double check this: it is weird that we instantiate the mutator twice in this test-case... And once with the default settings